clang/lib/DPCT/Rules.cpp

```diff
@@ -277,6 +277,25 @@ void registerPatternRewriterRule(MetaRuleObject &R) {
       R.BuildScriptSyntax, R.Priority));
 }
 
+void registerHelperFunctionRule(MetaRuleObject &R) {
+  if ((R.In == "get_default_queue" || R.In == "get_in_order_queue" ||
```

Comment:
Please emit warning "the rule is igored ,  the helperfunction kind rule is not support for functionality considration", if other helper function is specified.